config/locales/views/en.yml

```diff
@@ -0,0 +1,47 @@
+en:
+  application:
+    navigation_main:
+      dashboard: "Dashboard"
+    navigation_user:
+      sign_in: "Sign in"
+      sign_out: "Sign out"
+      sign_up: "Sign up"
+      edit_profile: "Edit profile"
+  dashboard:
+    index: 'Dashboard'
+  devise:
+    confirmations:
+      new:
+        title: "Resend confirmation instructions"
+    passwords:
+      new:
+        submit: "Send me reset password instructions"
+      edit:
+        update: "Update"
+    registrations:
+      new:
+        title: "Sign up"
+        legend_html: "Create your <strong>%{app_name}</strong> account"
+      edit:
+        update: "Update"
```

Comment:
There is a comment from @airatshigapov related to dict, I agree with it https://github.com/fs/rails-base/pull/198#issuecomment-54785971